doc/deployment/migrations.md

```diff
@@ -22,6 +22,8 @@ Note that the `pachctl migrate` command can be run either before or after you've
 
 Most importantly, you need to ensure that your cluster is "at rest" when you run `pachctl migrate`.  That is, there shouldn't be any ongoing activities that are changing the state of the cluster.  Examples would be running jobs or ongoing `put-file` requests.
 
+*Due to technical reasons, if you have a pipeline that specifies more than one environment variables in its pipeline spec, the first job that the pipeline spawns after the migration will re-process all input.* 
```

Comment:
As funny as "due to technical reasons" is, just reword to:

*Note: For v1.4 pipelines that specify environment variables, you will unfortunately need to reprocess the data for those pipelines as part of the v1.5 migration. This will automatically happen as part of the first job that spawns after the migration. Sorry for inconvenience.